task1.js

```diff
@@ -0,0 +1,6 @@
+
+// Write a program which reads a string from the standard input stdin, reverses it and then writes it to the standard output stdout.
+process.stdin.on("data", function(data) {
+    data = data.toString().split('').reverse().join('')
```

Comment:
We don't recommend to override the `data`, how about creating a new variable `const reversedStr = data.toString().split('').reverse().join('')`

Reply:
Yes, It is not a good habit to change the input parameter data, let me declare a variable